dist/nginx.conf

```diff
@@ -65,60 +65,55 @@ server {
   gzip_types text/plain text/css application/json application/javascript text/xml application/xml application/xml+rss text/javascript image/svg+xml image/x-icon;
 
   location / {
-    try_files $uri @proxy;
-  }
-
-  # If Docker is used for deployment and Rails serves static files,
-  # then needed must replace line `try_files $uri =404;` with `try_files $uri @proxy;`.
-  location = /sw.js {
-    add_header Cache-Control "public, max-age=604800, must-revalidate";
     add_header Strict-Transport-Security "max-age=63072000; includeSubDomains";
```

Comment:
afaik, Rails already sets HSTS headers… wouldn't this be an issue?

Reply:
In this case, these headers do not affect the `@mastodon` block